README.md

```diff
@@ -1,22 +1,103 @@
+   - Steam should open with a new **Console** Tab, go into it.
+   - Type `download_depot 504230 504233 1505052356460012099` into the Steam console and wait for it to install (there won't be a progress bar).
+   - Once complete, it will tell you where the files were installed. It will say something like this:
+   
+     ```bash
+     Depot download complete: "/Users/username/Library/Application Support/Steam/Steam.AppBundle/Steam/Contents/MacOS\\steamapps\\content\\app_504230\\depot_504233" (1259 files, manifest 1505052356460012099)
+     ```
+
+   - In this case, `depot_504233` is your Celeste folder that you should give to the build script. *This may vary.*
+   - Please move this folder to somewhere else, like your Downloads, or Documents folder, you can also rename it something like `Celeste-Linux`
+
+Once you have installed Celeste, make sure the folder contains content such as `Celeste.exe`, `Content` (folder), `Backups` (folder), `lib` (folder), `lib64` (folder).
+
+Also, Make sure the Name of this folder has no spaces in it, and make sure the path it's in has no spaces in it.
+
+
+## Building the IPA
+
+1. Open Terminal and navigate to where you'd like to install the build:
+   ```bash
+   cd Documents  # or your preferred directory
+   ```
+
+2. Clone the repository:
+   ```bash
+   git clone https://github.com/RoootTheFox/celeste-ios.git
+   ```
+
+      ****Make sure the path you clone into doesn't have any spaces in it, as this can cause problems***
+
+3. Navigate to the cloned directory:
+   ```bash
+   cd celeste-ios
+   ```
+
+4. Run the build script:
+   ```bash
+   ./build.sh
+   ```
+
+5. When prompted, drag and drop your celeste folder you downloaded earlier into the Terminal and press **Enter**.
+
+6. The build script will then ask:
+   - **"Do you want to enable a virtual game controller if no other input devices are detected? iOS 15+ only! (y/n)"**  
+     My recommendation is to type `n` for **no** unless you plan on using touch controls, as there could be issues with iOS 16 and above not auto-disabling the on-screen controller when a physical controller is connected.
+   
+   - **"Do you want to build MoltenVK as well? This is required for FNA3D, but building takes a while so it can be skipped on subsequent rebuilds. (y/n)"**  
+     If this is your first time building this, type `y` for **yes**
+
+## Completion
+
+After a while, the script will generate an IPA file called **celestemeow.ipa**, located in the folder where you cloned the project.
+
+You're now ready to install the IPA on your iOS device using something like TrollStore, AltStore or similar!
+* *Please note if you're using purchased certs from someone like KravaSign or Signulous, it is advised to use the website [KravaSigner](http://kravasigner.com) to sign your ipa's even if you didn't purchase your certs through KravaSign, apps like esign, feather and others can cause issues to do with rendered screen size*
```

Comment:
script already tells you this (minus the certificate files)

I don't want to link to any sites that i don't use/trust myself, although the issues with rendered screen size when signing the app in a specific way *are* a bug, and should maybe just mentioned with a small note like this:

> [!note]
> Apps like esign, feather and others can cause issues with rendered size, see #2

Reply:
Yeah I put this in partially because it felt like a nice way to finish it, and also because something along the same lines was in the original readme. but yeah that note looks really neat. I had no idea you could do that, very cool :)